packages/web-components/src/components/va-progress-bar/va-progress-bar.tsx

```diff
@@ -12,14 +20,19 @@ export class VaProgressBar {
   @Prop() enableAnalytics: boolean;
 
   /**
-   * The text label for the progress bar.
+   * Percent of progress made. 0 to 100.
    */
-  @Prop() label: string;
+  @Prop() percent: number = 0;
 
   /**
-   * Percent of progress made. 0 to 100.
+   * The text label for the progress bar.
    */
-  @Prop() percent: number;
+  @Prop({ mutable: true }) label: string = `${this.percent}% complete`;
+
+  @Watch('percent')
+  updateLabel(newPercent: number) {
+    this.label = `${newPercent}% complete`;
+  }
```

Comment:
Having a mutable prop with  `@Watch` looks like it might override a custom `label` value set by an engineer.

Reply:
You're correct. This isn't going to work.